test/pylib/scylla_cluster.py

```diff
@@ -431,7 +432,11 @@ def __init__(self, scylla_exe: str, replicas: int, test_base_dir: str,
         self.host_registry = host_registry
         self.artifact_registry = artifact_registry
         self.cluster: List[ScyllaServer] = []
+        self.started: Dict[str, ScyllaServer] = {}
+        self.stopped: Dict[str, ScyllaServer] = {}
+        self.removed: Set[str] = set()
```

Comment:
Stopped is a server that is there and can be restarted. Removed servers are gone. I planned to give a different error to tests if they try to start a removed server (instead of just a generic error for an unrecognized id). But I forgot to include this. I'll add it in the next version.